exporters/stdout/metric.go

```diff
@@ -44,10 +45,22 @@ type line struct {
 
 	Quantiles []quantile `json:"Quantiles,omitempty"`
 
+	Histogram []bucket `json:"histogram,omitempty"`
```

Comment:
I see below a switch on `e.config.PrettyPrint` which directly calls Fprintf, and I wonder if the structure would be more readable if the pretty-form of histogram printed in-line with the JSON, formatted within quotes, like:

```
... 
      Histogram: "
_pretty representation_
...
"
```

This could be implemented using a new type with a `MarshalJSON()` method that switched on `e.config.PrettyPrint` itself. How does this sound?